IceCubesApp/Resources/Localization/it.lproj/Localizable.strings

```diff
@@ -201,8 +201,12 @@
 "account.posts" = "Post";
 "account.relation.follows-you" = "Ti segue";
 "account.joined" = "Si è unito il giorno";
+<<<<<<< HEAD
```

Comment:
Conflict :)

Reply:
Fixed and commited. Thanks for catching it.